emission-book/public/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+  
```

Comment:
Favicon missing, I would probably get back the one `create-react-app` provides. 😄